CHANGELOG.md

```diff
@@ -0,0 +1,241 @@
+# Change Log for AWS IoT Embedded C SDK
+
+## 202009.00 (September 14, 2020)
+
+- The MQTT, JSON and AWS IoT Shadow service client libraries have been moved to the following individual repositories:
+   - [FreeRTOS/coreMQTT](https://github.com/FreeRTOS/coreMQTT) for MQTT client library
+   - [FreeRTOS/coreJSON](https://github.com/FreeRTOS/JSON) for JSON parser library
+   - [aws/device-shadow-for-aws-iot-embedded-sdk](https://github.com/aws/device-shadow-for-aws-iot-embedded-sdk) for AWS IoT Shadow service client library.
+- The above library folders in `libraries/` folder has been replaced with submodule to the individual repositories.
+
```

Comment:
We may have to mention about the Jobs library is not part of this release as it was part of the earlier release.

Reply:
Good suggestion.  Updated

Reply:
I think the word we used elsewhere is "refactored".  I think we also agreed to have it without the specific date, so just Sept 2020.

Reply:
can we combine line #11 and #7?  The libraries have been submoduled from their individual repos should be enough

Reply:
I have replaced `re-designed` with `re-factored`. 
I have combined the 2 bullet points mentioning about the new individual repositories and the use of submodule pointers

Reply:
Have removed the specific date to just use `Month Year` format